st2client/st2client/commands/inquiry.py

```diff
@@ -0,0 +1,140 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+
+from st2client.commands import resource
+from st2client.formatters import table
+from st2client.models.inquiry import Inquiry
+
+LOG = logging.getLogger(__name__)
+
+DEFAULT_SCOPE = 'system'
+
+
+class InquiryBranch(resource.ResourceBranch):
+
+    def __init__(self, description, app, subparsers, parent_parser=None):
+        super(InquiryBranch, self).__init__(
+            Inquiry, description, app, subparsers,
+            parent_parser=parent_parser,
+            commands={
+                'list': InquiryListCommand,
+                'get': InquiryGetCommand
+            })
+
+        # Registers extended commands
+        self.commands['respond'] = InquiryRespondCommand(self.resource, self.app,
+                                                         self.subparsers)
+
+        # Remove unsupported commands
+        # TODO: Refactor parent class and make it nicer
+        del self.commands['create']
+        del self.commands['update']
+
+
+class InquiryListCommand(resource.ResourceCommand):
+
+    # Omitting "schema" and "response", as it doesn't really show up in a table well.
+    # The user can drill into a specific Inquiry to get this
+    display_attributes = [
+        'id',
+        'roles',
+        'users',
+        'tag',
+        'ttl'
+    ]
+
+    def __init__(self, resource, *args, **kwargs):
+
+        self.default_limit = 20
```

Comment:
Any reason why `default_limit=20`? Can it be 50, to be consistent with other commands?

Reply:
I'm torn on this. On one hand I agree, consistency is a good idea. However, Inquiries are meant to be short-lived, relatively speaking. If the default limit is 50, then it would almost encourage a regular practice of balancing 50 concurrent executions that are waiting for a response. Even 20 is a bit extreme.

Reply:
I agree w/ @humblearner here. I prefer consistency.